api/controller.py

```diff
@@ -1643,7 +1642,7 @@ def contributor(
         feed = feed_class.page(
             _db=self._db, title=lane.display_name, url=url, lane=lane,
             facets=facets, pagination=pagination,
-            annotator=annotator, cache_type=CachedFeed.CONTRIBUTOR_TYPE,
+            annotator=annotator, cache_type=lane.CACHED_FEED_TYPE,
```

Comment:
If you're no longer going to use CONTRIBUTOR_TYPE, should it be removed from core?

Reply:
Oh sorry, that's in the Lane classes now.